deploy-test-liquidation-strategy/0_deploy.js

```diff
@@ -0,0 +1,212 @@
+const { promisify } = require('util');
+const exec = promisify(require('child_process').exec);
+const fs = require('fs');
+const path = require('path');
+
+module.exports = async ({ getNamedAccounts, deployments, ethers }) => {
+    const { deploy } = deployments;
+    const { deployer } = await getNamedAccounts();
+
+    const performanceFee = 1000; // 10% of gain
+
+    const genericAccountant = await deploy("GenericAccountant", {
+        from: deployer,
+        args: [performanceFee, deployer, deployer],
+        log: true,
+    });
+
+    const vaultPackage = await deploy("VaultPackage", {
+        from: deployer,
+        args: [],
+        log: true,
+    });
+
+    const factoryPackage = await deploy("FactoryPackage", {
+        from: deployer,
+        args: [],
+        log: true,
+    });
+
+    const factory = await deploy("Factory", {
+        from: deployer,
+        args: [factoryPackage.address, deployer, "0x"],
+        log: true,
+    });
+
+    // Compile the contracts
```

Comment:
You don't need that bro, when you run deployments scripts hardhat will auto compile the needed contracts for you.

Reply:
Oops, It's a remnant from the older deployment script which had to compile multiple times after changing address of the constant addresses. Thanks for pointing this out.